### ISSUE TITLE
Inaccessible Form Notifications Using alert()

### ISSUE THREAD

**Issue by GauravKarakoti** [GauravKarakoti/Omnifood#250]
# 🐛Describe the bug
The use of `alert()` for success/error messages creates accessibility issues:

- Not readable by screen readers
- Steals keyboard focus
- Provides no persistent visual feedback

**Comment by AkshayHaldar** [comment]
can you please add a ss ?
i give a ss please cross check 

![Image](https://github.com/user-attachments/assets/d5681733-ce43-4010-9ec8-56c0f30c1fb0)

**Comment by GauravKarakoti** [comment]
Yes the alert message is the current implementation, use a different way to provide success/error messages

**Comment by AkshayHaldar** [comment]
ok ,Gaurav i try to do it

**Comment by GauravKarakoti** [comment]
Go ahead @AkshayHaldar

**Comment by GauravKarakoti** [comment]
1 day left what's the progress

**Comment by vkoushik15** [comment]
@GauravKarakoti can you assign me this

**Comment by GauravKarakoti** [comment]
Go ahead @vkoushik15

**Comment by vkoushik15** [comment]
@GauravKarakoti hey i have made the notification like this it auto closes and doesnot maake the form inacesible check the photos
im going to send the pr

![Image](https://github.com/user-attachments/assets/658d1014-935c-4c4b-8355-4099f87109c8)
![Image](https://github.com/user-attachments/assets/707866b4-0e4c-4858-af3a-e33259e463bf)

**Comment by vkoushik15** [comment]
@GauravKarakoti  check out my pr

**Comment by GauravKarakoti** [comment]
> [@GauravKarakoti](https://github.com/GauravKarakoti) check out my pr

Heyyy did you raised the PR? i haven't recieved it and the PR accepting period is already over?
